app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+    
+    def logged_iin?
```

Comment:
This should be renamed to `logged_in?`

Reply:
yes this should be logged_in
sorry for the mis-typo